api/v1alpha1/storageclaim_types.go

```diff
@@ -44,6 +44,8 @@ type StorageClaimStatus struct {
 	Phase storageClaimState `json:"phase,omitempty"`
 }
 
+// TODO: remove after storageclassclaim migration controller and storageclassclaim is deleted
+
```

Comment:
This makes no sense, as this type is not used anymore in the code base. Why do we need to keep it?

Reply:
It is used in storageclassclaim, todo mentions that.

Reply:
Ha. ok. I understand now. Resolving.